caladrius/utils.py

```diff
@@ -171,33 +186,39 @@ def configuration():
     args = parser.parse_args()
 
     arg_vars = vars(args)
+    arg_vars["model_name"] = arg_vars["run_name"]
```

Comment:
model_name needed for interface or also other purpose? Maybe make a comment why you introduce this extra variable.

Reply:
I alter the run name without preserving the user's input for model name. In the interface, I have to split based on the convention of the run name. It is cleaner to save the model name input by the user regardless of where it is used.